src/main/java/com/hubspot/jinjava/el/ext/JinjavaListELResolver.java

```diff
@@ -30,12 +31,61 @@ public boolean isReadOnly(ELContext context, Object base, Object property) {
   @Override
   public Object getValue(ELContext context, Object base, Object property) {
     try {
+      // If we're dealing with a negative index, convert it to a positive one.
+      if (isResolvable(base)) {
+        int index = toIndex(property);
+        List<?> list = (List<?>) base;
+        if (index < 0) {
+          // Leave the range checking to the superclass.
+          index += list.size();
+        }
+        return super.getValue(context, base, index);
```

Comment:
we can just set `property = index + list.size()` and just have one `super.getValue()` outside the if